base/inc/widgets/base-slider.class.php

```diff
@@ -172,18 +172,18 @@ function render_template_part( $part, $controls, $frames ) {
 				?>
 				<ol class="sow-slider-pagination">
 					<?php foreach($frames as $i => $frame) : ?>
-						<li><a href="#" data-goto="<?php echo $i ?>"><?php echo $i+1 ?></a></li>
+						<li><a href="#" data-goto="<?php echo $i ?>" aria-label="<?php printf( __( 'open slide %s', 'so-widgets-bundle' ), $i+1 ) ?>"><?php echo $i+1 ?></a></li>
```

Comment:
"Open slide _x_" feels a bit ambiguous and might imply opening the slide's image or content somehow.

Reply:
What would you recommend instead?

Reply:
Perhaps "Show slide _x_" or "Display slide _x_"?

Reply:
I like both of those, but... hm... display sounds nicer to me.